SwiftJym/Views/ExercisesViews/ExercisesView.swift

```diff
@@ -83,7 +83,7 @@ struct SearchAndScanView: View {
                 TextField("Search exercises", text: $search)
             }
             .padding(.all, 15)
-            .background(Color.white)
+            .background(Color("bg"))
```

Comment:
the point of the dark scheme for 'bg' was to enable its use without having to worry about the colorScheme